ingestclient/client.py

```diff
@@ -69,6 +68,7 @@ def worker_process_run(api_token, job_id, pipe, config_file=None, configuration=
         pipe(multiprocessing.Pipe): the receiving end of the pipe that communicates with the master process.
         config_file(str): the path to the configuration file (configuration required if omitted)
         configuration(Configuration): a pre-loaded configuration object (config_file required if omitted)
+        run_complete (bool): Run ingest completion when done
```

Comment:
Whats the logic here?  Is there a reason we might not want to run complete?

Reply:
Hmm, now we probably don't need this anymore. Should I remove now or wait until someone other than me does some additional testing?